chainer/backends/__init__.py

```diff
@@ -0,0 +1,37 @@
+import numpy
+
+import chainer
+from chainer.backends import cuda
+from chainer.backends import intel64
+
+
+def copyto(dst, src):
+    """Copies the elements of an ndarray to those of another one.
+
+    This function can copy the CPU/GPU arrays to the destination arrays on
+    another device.
+
+    Args:
+        dst (numpy.ndarray or cupy.ndarray): Destination array.
+        src (numpy.ndarray or cupy.ndarray): Source array.
```

Comment:
Could you add `intel64.mdarray` to types of args?

Reply:
Fixed.